src/pkgcheck/bash/__init__.py

```diff
@@ -87,13 +91,12 @@ def build_library(output_path, repo_paths):  # pragma: no cover
     from .. import _const
 except ImportError:  # pragma: no cover
     # build library when running from git repo or tarball
-    if not os.path.exists(lib) and 'tree-sitter-bash' in os.listdir(const.REPO_PATH):
+    if not syslib and not os.path.exists(lib) and 'tree-sitter-bash' in os.listdir(const.REPO_PATH):
```

Comment:
```suggestion
    if syslib is None and not os.path.exists(lib) and 'tree-sitter-bash' in os.listdir(const.REPO_PATH):
```

I think in general we prefer explicitly checking for `None` even no other valid value would evaluate false.